<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SymbolTableDialog</class>
 <widget class="QDialog" name="SymbolTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>edit symbol table</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <property name="spacing">
    <number>16</number>
   </property>
   <item row="0" column="0" rowspan="7" colspan="4">
    <widget class="QTableWidget" name="tableWidget">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>300</number>
     </attribute>
     <column>
      <property name="text">
       <string>key</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>value</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="delButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>delete</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QPushButton" name="mergeWithButton">
     <property name="minimumSize">
      <size>
       <width>85</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>merge with</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="loadButton">
     <property name="text">
      <string>load</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>save</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="defaultButton">
     <property name="text">
      <string>default</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SymbolTableDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SymbolTableDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
